### ISSUE TITLE
iOS 13.3

### ISSUE THREAD

**Issue by basis2000** [Matchstic/Xen-HTML#165]
The Lockscreen widgets are showing only after unlock the device (faceID)
iPhone X
iOS 13.3

**Comment by batfirex2** [comment]
For me, iPhone 6s, LS widget only shows up when fingerprint is read. 

Also this next one is a sporadic issue. when unlocking phone and quickly swiping the page, springboard crashes. Crashreporter states its cylinder and xenhtml. 

Lastly. I feel the widgets take longer to refresh the data they show when compared to 13.2.3.

**Comment by kevinbish** [comment]
I am having the same issue. 
"The Lockscreen widgets are showing only after unlock the device." (Touch ID)
iPhone 7 Plus 
iOS 13.3

**Comment by m7amed3la2** [comment]
The Lockscreen widgets are showing only after unlock the device (faceID)
iPhone X
iOS 13.3
When it was workes perfectly with me in ios 13.2.3

**Comment by DroppedHeart** [comment]
I've got the same issue, iPad Mini 4, iOS 13.3. Can't comment on how it worked prior as this is my first time using the tweak.

**Comment by Matchstic** [comment]
Duplicate of #156
